libenkf/src/model_config.c

```diff
@@ -634,3 +642,215 @@ void model_config_fprintf_config( const model_config_type * model_config , int e
   fprintf(stream , "\n\n");
 
 }
+
```

Comment:
In a similar manner that site_config knows how the site configuration file should be loaded, the model_config now knows how the model configuration file should be loaded. This is basically just code moved from enkf_main.c to model_config.c together with the new method model_config_alloc_content.